components/post_view/acknowledgements/post_acknowledgements.scss

```diff
@@ -0,0 +1,73 @@
+@charset "UTF-8";
+
+@import 'utils/mixins';
+@import 'utils/variables';
+
+.AcknowledgementButton {
+    @include button-xsmall;
+
+    position: relative;
+    display: flex;
+    min-width: 0;
+    align-items: center;
+    border: 0;
+    background: rgba(var(--denim-status-online-rgb), 0.08);
```

Comment:
use variable `--online-indicator`

Reply:
please `--online-indicator` in place of all instances of `--denim-status-online`